include/d/actor/d_a_tpota.h

```diff
@@ -2,22 +2,36 @@
 #define D_A_TPOTA_H
 
 #include "f_op/f_op_actor.h"
+#include "d/d_particle.h"
```

Comment:
You should be able to move this include to the .cpp to avoid it going in the header https://github.com/zeldaret/tww/blob/main/docs/coding_guidelines.md#includes